shared/langs/en-us/standards/standards-cc-math-dictionary.json

```diff
@@ -412,8 +412,9 @@
     ]
   },
   "6.SP.A.2": {
-    "description": "Understand that a set of data collected to answer a statistical question has a distribution which can be described by its center, spread, and overall shape.",
+    "description": "Understand that a set of data collected to answer a statistical question has a distribution which can be described by its center, spread, overall shape, or a frequency table.",
```

Comment:
fascinating. are the standards a moving target or did we miss this when we created the dictionary?